src/pages/Delivery/Delivery.styled.jsx

```diff
@@ -24,10 +24,9 @@ const HeaderMap = styled.div`
 `
 const HeaderText = styled.div`
   text-align: center;
-  display: flex;
-  flex-direction: column;
-  margin-left: 25px;
   height: auto;
+  position: absolute;
+  left: 40% ;
```

Comment:
If HeaderText have position absolute, the father should have position relative. Here's some documentation about it https://css-tricks.com/absolute-positioning-inside-relative-positioning/